CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## next
+
+### NEW - option `--output-name`
+
+It is possible to specify which constant or function you would like to output by adding `--output-name=myCustomConstantOrFunction`.
+
+### NEW - option `--project-dir`
+
+Adding `--project-dir=/path/to/project` helps elm pick correct `elm-package.json` and `elm-stuff` if the file to run is in a subdirectory of a project.
```

Comment:
Add `--project-dir=/path/to/project` to help Elm pick the correct `elm-package.json` and `elm-stuff` if the Elm file is in a subdirectory.